src/app/parcels/ActionBar/ActionModals/DateChangeModal.tsx

```diff
@@ -13,13 +13,61 @@ import dayjs, { Dayjs } from "dayjs";
 import { getStatusErrorMessageWithLogId } from "../Statuses";
 import { getDbDate } from "@/common/format";
 import { getUpdateErrorMessage, packingDateOrSlotUpdate } from "./CommonDateAndSlot";
+import { ParcelsTableRow } from "@/app/parcels/parcelsTable/types";
+
+interface ContentProps {
+    onClose: () => void;
+    onDateSubmit: (event: React.FormEvent<HTMLFormElement>) => void;
+    setDate: (date: Dayjs) => void;
+    selectedParcels: ParcelsTableRow[];
+    maxParcelsToShow: number;
+    warningMessage: string;
+}
+
+const DateChangeInput = React.forwardRef<HTMLInputElement, DateInputProps>(
+    (props, elementToFocusRef) => {
+        return (
+            <>
+                <Heading>What date would you like to change to?</Heading>
+                <SingleDateInput setDate={props.setDate} ref={elementToFocusRef} />
+            </>
+        );
+    }
+);
+
+DateChangeInput.displayName = "DateChangeInput";
+
+const DateChangeModalContent: React.FC<ContentProps> = ({
+    onClose,
+    onDateSubmit,
+    setDate,
+    selectedParcels,
+    maxParcelsToShow,
+    warningMessage,
+}) => {
+    const elementToFocusRef = useRef<HTMLInputElement>(null);
+
+    useEffect(() => {
+        elementToFocusRef.current?.focus();
+    }, []);
```

Comment:
Can we name this for what it is, i.e., `dateChangeInputRef`? It's pretty clear it's used to focus the element, would be nice to know which element at a glance. This is both here and elsewhere :)